sda-commons-server-kafka/src/test/java/org/sdase/commons/server/kafka/config/KafkaPropertiesSpec.java

```diff
@@ -26,4 +27,82 @@ public void itShouldBuildSaslStringCorrectly() {
         "org.apache.kafka.common.security.plain.PlainLoginModule required username='user' password='password';";
     assertThat(props.getProperty("sasl.jaas.config")).isEqualTo(saslJaasConfig);
   }
+
+  @Test
+  public void itShouldConfigureSecurityForAdminConfig() {
```

Comment:
Will classes ending with "Spec" be picked up by Junit at all?

Reply:
Yes. JUnit is only interested in the `@Test` Though I agree that the name of the file could be improved to match our usual pattern.